hazelcast/config.py

```diff
@@ -101,20 +132,14 @@ def __init__(self):
         self.lifecycle_listeners = []
         """ Lifecycle Listeners, an array of Functions of f(state)"""
 
-        self.near_cache_configs = {}  # map_name:NearCacheConfig
-        """Near Cache configuration which maps "map-name : NearCacheConfig"""
-
-        self.flake_id_generator_configs = {}
+        self.flake_id_generators = {}
         """Flake ID generator configuration which maps "config-name" : FlakeIdGeneratorConfig """
 
-        self.serialization_config = SerializationConfig()
-        """Hazelcast serialization configuration"""
-
         self.logger_config = LoggerConfig()
```

Comment:
Is there any special reason why we left `_config` name in `logger_config` while we have removed it from all others?

Reply:
Possibly leftover, fixed in #219